ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewManagerPropertyUpdater.java

```diff
@@ -13,7 +13,7 @@
 
 public class ViewManagerPropertyUpdater {
   public interface Settable {
-    Map<String, String> getProperties();
+    Map<String, String> getProperties(Map<String, String> map);
```

Comment:
Haha, all i thought was make smallest change to the code.
Maybe i should write like the official android api:
DisplayMetrics dm = new DisplayMetrics();
getWindowManager().getDefaultDisplay().getMetrics(dm);

is this ok?